packages/components/src/font-size-control/font-size-control-styles.js

```diff
@@ -1,8 +1,25 @@
-import { css } from '@wp-g2/styles';
+import { css, ui } from '@wp-g2/styles';
 
 export const FontSizeControl = css`
 	appearance: none;
 	border: none;
 	margin: 0;
 	padding: 0;
 `;
+
+export const SelectDropdownWrapper = css`
+	width: calc(100% * 0.5 - (${ui.space(2)} * 2));
```

Comment:
I really don't like that we have to do this.

I don't see a way around it, unless we develop some sort of intelligent Grid inspired `templateColumn` feature for `HStack`.